configs/engine/rendering.cfg

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Unstage this file. In fact, if you don't mind, add it to gitignore.

(this file is a side-effect of a PR of mine that got merged recently, and i forgot to gitignore this)